base_cron_exclusion/README.rst

```diff
@@ -0,0 +1,65 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
see many remarks from https://github.com/OCA/server-tools/pull/1392#pullrequestreview-164502064 that might apply here